frontend/src/layout/TopContent/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { LatestVersion } from './LatestVersion'
+import { User } from './User'
+import { WorkerStats } from './WorkerStats'
+import { CommandPaletteButton } from './CommandPaletteButton'
+import { isMobile } from 'lib/utils'
+
+import './index.scss'
+
+export function TopContent(): JSX.Element {
+    return (
+        <div className="content py-3 layout-top-content">
+            <div
+                className="layout-top-content"
+                style={{
+                    display: 'flex',
+                    flexDirection: 'row',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    fontSize: 13,
+                }}
+            >
+                {!isMobile() && <CommandPaletteButton />}
```

Comment:
The button won't show up, since it's essentially impossible to detect such details. But the keyboard shortcut will work regardless of that. The button is just a suggestion.